custom-plugin-rules/default-enum-match/README.md

```diff
@@ -0,0 +1,97 @@
+# Default is an Enum
+
+Authors:
+- [@hawkeyexl](https://github.com/hawkeyexl) Manny Silva ([Doc Detective](https://doc-detective.com))
+
+## What this does and why
+
+This rule makes sure that when a schema has both `default` and `enum` properties, the `default` property is set to one of the values in the `enum` array. This is useful for API documentation where you want to make sure that the default value is valid.
```

Comment:
Wow, I'm surprised this one isn't a built-in rule. Looks like this is the closest one: https://redocly.com/docs/cli/rules/no-enum-type-mismatch

I think this would be a good candidate for a built-in rule though. What do you think @lornajane ?

Reply:
Definitely! Let's get it in the cookbook first though